README.md

```diff
@@ -1,2 +1,242 @@
-# learn-user-testing
-Learn how to test your app with real humans 
+# Learn User Experience Testing
+Learn how to test your app with real humans
+
+## Why?
+This is _probably_ the most important/useful skill any "_creative technologist_"
+has. 💪
+
+Well conducted user experience testing helps you ensure that what you are
+planning to build is going to solve the problem for your users that you set out
```

Comment:
UX tests are not _just_ used in the _planning_ phase, they are done with prototypes and "finished" apps. 
Ideally ideas for UX should be tested as _early_ as possible to avoid wasting time building _ineffective_ UX (_that nobody wants or will use_).

Reply:
Proposed rephrase:
> Well conducted user experience testing helps you ensure at each stage of 
development that what you are planning to build is going to solve the problem 
for your users that you set out to solve. ✅